src/index.js

```diff
@@ -7,6 +7,9 @@ import {getOptions} from 'loader-utils'
  * @returns {string} the code needed to handle the riot hot reload
  */
 function hotReload(path) {
+  if (process.platform === 'win32') {
```

Comment:
Please avoid platform hacks and try to use the native `path` methods like suggested here https://github.com/riot/webpack-loader/pull/26#issuecomment-489450583

Reply:
Path.normalize doesn't solve the problem, but the if-statement is probably not needed.